src/styles/DiscussionCard.css

```diff
@@ -1,23 +1,62 @@
 .card {
   border-radius: 5px;
   background-color: #fff;
-  padding: 2.5% 2.5% 5% 2.5%;
+  padding: 1% 3% 2% 3%;
   margin-bottom: 2.5%; }
 
+.card-title-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center; }
+
 .card-title {
   text-align: left;
-  margin: 0; }
+  margin: 0;
+  display: inline-block;
+  font-size: 115%; }
+
+.card-tag {
+  border-radius: 5px;
+  display: inline-block;
+  padding: 4px 6px;
+  color: #fff;
+  font-size: 90%; }
+
+.R {
```

Comment:
We can probably do some refactoring with classnames and combine this styling with the styling I did to render the domain names.